libs/src/api/ServiceProvider.ts

```diff
@@ -74,7 +74,7 @@ export class ServiceProvider extends Base {
     const timings = await timeRequests({
       requests: creatorNodes.map((node) => ({
         id: node.endpoint,
-        url: `${node.endpoint}/health_check/verbose`
+        url: `${node.endpoint}/health_check`
```

Comment:
it's okay to make this change because the verbose endpoint is still around and still returns the same result as it is currently. previously i think we had tried to remove verbose and libs threw errors cause clients updated faster than SPs

Reply:
yep exactly